test/tools/TestExe/TestExe.cs

```diff
@@ -40,6 +47,48 @@ static void EchoArgs(string[] args)
             }
         }
 
+        static void WriteStandardOutput()
+        {
+            // write a string of characters (bytes), this closest resembles a binary
+            // write the string "test" with an accent over the e
+            byte[] testbytes = new byte[] { 116, 233, 115, 116 };
```

Comment:
Why not just write all the bytes from 0 to 255?

Reply:
no need
[char]233 is all that is needed to show the problem, and is readable from the command line if you run it.